src/main/java/org/comic_conmuseum/fan_forge/backend/models/Exhibit.java

```diff
@@ -44,7 +47,8 @@ public Exhibit (ResultSet rs, @SuppressWarnings("unused") int rowNum) throws SQL
     public Instant getCreated() { return created; }
     public String[] getTags() { return tags; }
     public Artifact getCover() { return cover; }
-    
+    public boolean getFeatured() { return featured; }
```

Comment:
Java conventions suggest that this should be `isFeatured`, not `getFeatured`

Reply:
Good idea, done